### DISCUSSION TITLE
Kitty stopped split pane to wider direction

### DISCUSSION THREAD

**Discussion by osenvosem** [kovidgoyal/kitty#7059] [Q&A]
At some point Kitty split pane to wider direction. Before that Kitty would create a vertical split if current space is larger horizontalli and horizontal split if the space is lager vertically.

With this configuration:

```
map cmd+enter launch --location=split --cwd=current
enabled_layouts splits:split_axis=split
```

**Comment by kovidgoyal** [comment]
I dont really understand what you are saying, but I will note that the split_axis option has valid values only horizontal and vertical not "split"

**Reply by osenvosem** [reply]
The previous behaviour was: if width of current window is greater than height then Kitty created vertical split and horizontal if otherwise. How can I configure this behaviour?

**Reply by kovidgoyal** [reply]
That is still the current behavior with --location=split

**Reply by osenvosem** [reply]
But I have this option set and it splits incorrectly. Maybe I don't understand something. 🤷‍♂️

<img width="1792" alt="Screenshot 2024-01-25 at 12 15 09" src="https://github.com/kovidgoyal/kitty/assets/6762673/09e75989-5464-423a-b1a5-cb893f64f55c">

**Reply by kovidgoyal** [reply]
kitty --config NONE -o enabled_layouts=splits -o 'map f1 launch --location=split'

**Reply by osenvosem** [reply]
Hmmm... this works as expected

**Reply by osenvosem** [reply]
If I change `cmd+enter` to something else, it works but not with `cmd+enter`

**Comment by kovidgoyal** [comment]
#7016

**Reply by osenvosem** [reply]
Thank you
